app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -97,46 +99,53 @@ living costs, to train (depending on your immigration status, you may get a redu
 
 ## Apply for teacher training
 
-We can help you make an application for teacher training in England. There are a number of steps to follow.
+We can help you make an application for teacher training in England. Sign up for a teacher training adviser to get help right away and throughout the application process.
```

Comment:
@laurahermionetennant92 just checking - is 'right away' accurate? How long does it take to get assigned an adviser/to get help?

Reply:
Hmm good question, just checking with the team

Reply:
Worst case scenario is 48 hours so I think this is OK